application/provider.go

```diff
@@ -105,6 +106,12 @@ func NewProvider(prefix string, scopes ...string) (*ProviderImpl, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to create logger")
 	}
+	if os.Getenv("POD_NAMESPACE") == "default" || os.Getenv("TIDEPOOL_USE_DEVLOG") != "" {
```

Comment:
A few things here:
- I would highly prefer we not force this on all local developers (namely me). Either remove this code from the PR (and keep in a local stash for your use) or just use an environment variable to enable/disable. That is, remove the `POD_NAMESPACE` check at a minimum.
- If you're going to use an environment variable, consider using the `configReporter` to fetch it. That is the generally the standard for configuration in platform. Yea, I know, consistency.
- If you're going to use an environment variable, please use if/else to create one or the other logger. That is, don't create the JSON logger and then replace it with the plain text logger. Just create one or the other.

Reply:
Doh, sorry, it was my intention to make this opt-in, not opt-out. I was exploring ways of getting it enabled and pushed this in a bit of a rush (I was hacking on it during the all-hands :D)

I'll definitely get it to be opt-in only... I'll have to explore more options.

The TIDEPOOL_USE_DEVLOG is great, but is more intended for folks like @Roukoswarf that run platform directly. Those of us using development/k8s can't easily inject env vars into the development environment (that I'm aware of).

That said, I don't want to add an env var to the helm chart that's only use is for local development... that doesn't feel great. 

So I think my next step is to poke around in development and see if there's some kind of easy to inject local stuff easily... Opt-in of course.

Reply:
If you know of a way to do that (what I mentioned above about adding local config bits to development), I'm all ears.

Reply:
Won't something like `kubectl set env deployment/data TIDEPOOL_USE_DEVLOG=true` work?

Reply:
It does... though I was hoping to be able to set something once and have it be "always on".

If I add a setting to the helm chart, I can achieve that. What would you think about adding this to the helm chart?:
```
 - name: TIDEPOOL_LOGGER_PACKAGE
   value: {{ .Values.global.loggerPackage | default "" }}
```
This would allow us to set a value in development/local/Tiltconfig.yaml:
```
global:
  loggerPackage: "devlog"
```
If the value of TIDEPOOL_LOGGER_PACKAGE is anything other than "devlog", it would use the default (json) logger.

100% opt-in. Safe from accidentally being set in non-local dev environments, and still allows those that want to (myself among them) to "set it once and forget it".

Reply:
I've put up a PR: https://github.com/tidepool-org/development/pull/282

Reply:
Okay, I can live with that.